homeassistant/components/tautulli/sensor.py

```diff
@@ -56,84 +73,43 @@ async def async_setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Create the Tautulli sensor."""
-
-    name = config[CONF_NAME]
-    host = config[CONF_HOST]
-    port = config[CONF_PORT]
-    path = config[CONF_PATH]
-    api_key = config[CONF_API_KEY]
-    monitored_conditions = config.get(CONF_MONITORED_CONDITIONS, [])
-    users = config.get(CONF_MONITORED_USERS, [])
-    use_ssl = config[CONF_SSL]
-    verify_ssl = config[CONF_VERIFY_SSL]
-
-    session = async_get_clientsession(hass=hass, verify_ssl=verify_ssl)
-
-    api_client = PyTautulli(
-        api_token=api_key,
-        hostname=host,
-        session=session,
-        verify_ssl=verify_ssl,
-        port=port,
-        ssl=use_ssl,
-        base_api_path=path,
+    _LOGGER.warning(
+        "Tautulli yaml config with host %s has been imported. Please remove it",
```

Comment:
Suggest more information message.

```suggestion
        "Configuration of the Tautulli platform in YAML is deprecated and will be "
        "removed in Home Assistant 2022.6; Your existing configuration for host %s"
        "has been imported into the UI automatically and can be safely removed "
        "from your configuration.yaml file",
```